middleware_defaults.go

```diff
@@ -6,6 +6,9 @@ import (
 	"github.com/riverqueue/river/rivertype"
 )
 
+// MiddlewareDefaults should be embedded on any middleware implementation. It
+// helps identify a struct as middleware, and guarantee forward compatibility in
```

Comment:
Might be slightly more readable as "guarantees":

```suggestion
// helps identify a struct as middleware, and guarantees forward compatibility in
```

Reply:
Good catch. Thanks.